### DISCUSSION TITLE
Unable to unregister ssh host

### DISCUSSION THREAD

**Discussion by markfraser** [smallstep/certificates#806] [Q&A]
I am unable to unregister ssh hosts from the inventory. When I run step-ssh-ctl unregister I get -
error reading /root/.step/config/identity.json: open /root/.step/config/identity.json: no such file or directory
This file doesn't appear to be created by bootstrapping the ca. If I try to force a new host ssh cert using the AWS Web Services provisioner token I get - error="authority.Authorize: authority.authorizeSSHSign: authority.authorizeToken: token already used"
Is there a way to remove a host from inventory and revoke it's cert on the CA?
I also tried running 'https://files.smallstep.com/ssh-host-uninstall.sh', which fails to unregister, fails to deactivate (same lack of identity.json file) but then proceeds to delete /etc/ssh/ssh_host_ecdsa_key-cert.pub. Now the host is part of the inventory but the cert has been deleted and can't generate a new one.

**Comment by maraino** [comment]
You can try to generate a certificate for that host, with the same host-id if possible using for example:

```console
step ssh certificate --host --host-id <uuid> --token <token> 
```

This will create the identity certificate. And then try to unregister in the normal way. If that does not work please contact support@smallstep.com
